### PR TITLE
Update case study and use consistent wording

### PR DESCRIPTION
**Pull request by jlarfors**

### CODE REVIEW

File: src/posts/cases/value-stream-mapping-with-visma.md [hunk 1]
```diff
@@ -36,25 +36,25 @@ Part of their research involved a call with Verifa, who proposed a Continuous De
...
+Verifa held a two day Value Stream Mapping workshop with the relevant stakeholders at Visma Software Oy to create a value stream of the current delivery process (from a workable item to deployed in production). The value stream was then annotated with “waste” cards, to highlight inefficiencies and these waste cards were weighted by their perceived impact on the actors. These waste cards were discussed during the workshop to help build a common understanding and discover if they are a symptom or a problem.
 
 <blockquote>
-  ‘The Value Stream Map clearly visualises our delivery process (and its pains). Verifa helped lead discussions around many topics which was useful in gaining a common understanding of what we can do to improve.’
+  ‘The value stream clearly visualises our delivery process (and its pains). Verifa helped lead discussions around many topics which was useful in gaining a common understanding of what we can do to improve.’
```
[the remark this answers is not in the document]
Comment (jlarfors) [reply src/posts/cases/value-stream-mapping-with-visma.md:45]: I kinda agree, I capitalized it for every use now.

File: src/posts/cases/value-stream-mapping-with-visma.md [hunk 2]
```diff
@@ -36,25 +36,25 @@ Part of their research involved a call with Verifa, who proposed a Continuous De
...
 
-Verifa held a two day workshop with the relevant stakeholders at Visma Software Oy to create a VSM of the current release process (from a workable item to deployed in production). The VSM was then annotated with “waste” cards, to highlight inefficiencies and these waste cards were weighted by their perceived impact on the actors in the VSM. These waste cards were discussed during the workshop to help build a common understanding and discover if they are a symptom or a problem.
+Verifa held a two day Continuous Delivery workshop with the relevant stakeholders at Visma Software Oy to create a Value Strean of the current delivery process (from a workable item to deployed in production). The Value Stream was then annotated with “waste” cards, to highlight inefficiencies and these waste cards were weighted by their perceived impact on the actors. These waste cards were discussed during the workshop to help build a common understanding and discover if they are a symptom or a problem.
```
Comment (kvarak) [inline src/posts/cases/value-stream-mapping-with-visma.md:42]: Value Strean -> Value Stream

File: src/posts/cases/value-stream-mapping-with-visma.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kvarak) [reply src/posts/cases/value-stream-mapping-with-visma.md:45]: Still in lower case?

File: src/posts/cases/value-stream-mapping-with-visma.md [hunk 3]
```diff
@@ -36,25 +36,25 @@ Part of their research involved a call with Verifa, who proposed a Continuous De
...
   <br/>
   <cite>- Olli Suihkonen, Senior Manager, Visma Software Oy.</cite>
 </blockquote>
 
 ## Results: a clear path forward
 
-After the workshop, Verifa compiled a report highlighting the findings and including some of Verifa’s own assessments. Several of the findings were symptoms of a common problem, and this helps prioritise and focus work on addressing the things that matter. Together with the Value Stream Map, the report provides a lot of useful material for Visma Software Oy to continue their Continuous Delivery journey and Verifa will continue to assist as needed.
+After the workshop, Verifa compiled a report highlighting the findings and including some of Verifa's own assessments. Several of the findings were symptoms of a common problem, and this helps prioritise and focus work on addressing the things that matter. Together with the value stream, the report provides a lot of useful material for Visma Software Oy to continue their Continuous Delivery journey and Verifa will continue to assist as needed.
```
Comment (kvarak) [inline src/posts/cases/value-stream-mapping-with-visma.md:52]: value stream in lower case

**Review by kvarak — APPROVED** [review]
Except for some comments it LGTM

File: src/posts/cases/value-stream-mapping-with-visma.md (hunk 3 above)
Comment (jlarfors) [reply src/posts/cases/value-stream-mapping-with-visma.md]: Thanks! Done.

File: src/posts/cases/value-stream-mapping-with-visma.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jlarfors) [reply src/posts/cases/value-stream-mapping-with-visma.md:46]: Thanks! Done.

**Review by andreaslarfors — APPROVED** [review]

**Pull request merged**